tests/release_tests/test_launch/launch-agent-patch.yaml

```diff
@@ -0,0 +1,39 @@
+metadata:
+  name: launch-agent-release-testing
+  namespace: wandb-release-testing
+spec:
+  template:
+    spec:
+      containers:
+        - name: launch-agent
+          image: wandb-launch-agent:latest
```

Comment:
So we are always using the latest image? I feel like this should build and agent image from the current branch and deploy that to the cluster